README.md

```diff
@@ -1 +1,23 @@
-# company-search-consumer
\ No newline at end of file
+# company-search-consumer
+A service that consumes messages from the stream-company-profile topic and then sends a RESTful POST request to the search.api.ch.gov.uk service. The data sent through the POST request will be used in search.api.ch.gov.uk to maintain and update the elastic search alpha_index.
```

Comment:
This summary will be incorrect if the functionality of search.api.ch.gov.uk changes.